third_party/modules/xla/metadata.json

```diff
@@ -17,6 +17,7 @@
         "20250103.0-5f1fe6a",
         "20250122.0-cc075be",
         "20250123.0-cc075be",
+        "20250124.0-cc075be"
```

Comment:
Usually we use the commit's date, and add `.1, .2` etc.. to notify incremental updates

Reply:
Done. I initially followed what was done for 0123 which didn't include .1